### DISCUSSION TITLE
How to compute Terminal Set for MPC without MPT?

### DISCUSSION THREAD

**Discussion by ClaraKowalczyk** [yalmip/YALMIP#1371] [Optimization modelling]
Hello,
I was wondering if there exists tools (in Yamilp or other optimization toolbox) to compute the Terminal Constraints Set X_f of an MPC (linear and discrete-time), without having to use mpt_infset from the Multi Parametric Toolbox.
Any suggestions ? :)

**Comment by johanlofberg** [comment]
Of course, it is just polytopic geometry/linear programming.

**Reply by johanlofberg** [reply]
But not built-in to YALMIP, that's the whole purpose of MPT to be a layer outside
